src/pijama_ast/src/lib.rs

```diff
@@ -129,31 +165,45 @@ impl<'a> Display for Primitive {
     }
 }
 
-/// Encapsulates a conditional where "if `cond` then `body`"
+/// Encapsulates a conditional statement in Pijama's syntax of the 
+/// form "if `cond` then `body`".
 #[derive(Debug, Eq, PartialEq)]
 pub struct Branch<'a> {
+    /// The conditional part of the Branch that is checked for truthiness.
     pub cond: Located<Block<'a>>,
+    /// The body of the Branch that is executed if `cond` is truthy.
     pub body: Located<Block<'a>>,
 }
 
+/// A [`Node`] in the AST that encapsulates the different
+/// expressions and statements that Pijama's syntax supports.
```

Comment:
I'd say that the only statements here are `FnDef` and `LetBind`, the other variants should be called expressions because they return something.